src/main/webapp/resources/js/components/metadata/TargetMetadataRestriction.tsx

```diff
@@ -106,17 +117,22 @@ export function TargetMetadataRestriction({
       visible={tooltipVisible}
     >
       <Form.Item {...feedback} style={{ marginBottom: 0 }}>
-        <Select
-          style={{ width: `100%` }}
+        <Radio.Group
+          style={{ display: "flex", width: "100%" }}
           value={field.restriction}
-          onChange={(value) => onChange({ ...field }, value)}
+          onChange={({ target: { value } }) => onChange({ ...field }, value)}
```

Comment:
JSX props should not use arrow functions, instead, create the same function instead a useCallback

Reply:
Don't worry about these ones.  It's not as performant as wrapping the function in a `useCallback` and I was getting warnings from sonarlint, but in the most recent update, those seem to have gone away.  Let's just leave these.